src/analysis/AnalysisClient.js

```diff
@@ -157,8 +199,15 @@ class AnalysisClient {
         that.analyzedProductsInfo.push(analyzedProduct);
       } 
     });
+
+    let categorizedItems = this._filterProductsToCategories(that.analyzedProductsInfo);
+    
+    //write all data into separate file for each category
+    this._writeAllCategories(categorizedItems);
+
+
     //write price analysis info for analyzed products
-    this._writeToFile("analyzed_items_info.txt");
+    //this._writeToFile("analyzed_items_info.txt");
```

Comment:
Is there a reason you commented this out?

Reply:
I forgot to delete it after testing.  Fixed here  9aec7e